src/css/components/search.scss

```diff
@@ -0,0 +1,56 @@
+@import "../core.scss";
+
+$search-btn-width:   4.5rem;
+$search-btn-height:  2.0rem;
+
+@mixin search-icon {
+  background-image: url('../img/i-search.svg');
```

Comment:
Can you use the `$static-img-path` variable here. Example in https://github.com/18F/cg-style/blob/master/src/css/base/background.scss#L15